posts/2019-12-30-reducing-support-for-32-bit-apple-targets.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: "Reducing support for 32-bit Apple targets"
+author: Pietro Albini
+---
+
+The Rust team is sad to announce that Rust 1.41.0 (to be released on January
```

Comment:
I don't think we need to make an emotional statement. Either of these would be fine, I think:

Reply:
```suggestion
The Rust team regrets to announce that Rust 1.41.0 (to be released on January
```

Reply:
Or, with no "padding" at all:

```suggestion
Rust 1.41.0 (to be released on January
```